packages/@aws-cdk/aws-lambda-event-sources/lib/kafka.ts

```diff
@@ -22,6 +23,13 @@ export interface KafkaEventSourceProps extends BaseStreamEventSourceProps {
    * @default none
    */
   readonly secret?: secretsmanager.ISecret
+  /**
+   * The identifier for the Kafka consumer group to join. The consumer group ID must be unique among all your Kafka event sources. After creating a Kafka event source mapping with the consumer group ID specified, you cannot update this value.
+   * @see https://docs.aws.amazon.com/lambda/latest/dg/with-msk.html#services-msk-consumer-group-id
+   *
+   * @default - none
+   */
+  readonly consumerGroupId?: EventSourceMappingOptions['kafkaConsumerGroupId'];
```

Comment:
Ultimately this resolves to a string so I'm not sure what the benefit of assigning the type this way.

Reply:
Yes it is just a string, but my intention was to give someone that is looking at the code some guidance that the consumerGroupId is the same as the kafkaConsumerGroupId. And then it is easier to follow the path of the property to further documentation and usage. In this case the consumerGroupId is just a renaming of the property kafkaConsumerGroupId to have a shorter property name because we are aöready in the context of kafka properties.
But I get that this way of defining properties is a new style and therefore not optimal. 
Please let me know if I should change it to or the property path reasoning has more value than just putting string here.

Reply:
I appreciate trying to give the extra context but I'm not sure that writing it this way will play nice with JSII when translating it into the other languages we use. That, and consistency in style is something I tend to think is pretty important for those who come after and try to understand the code to make changes again. I would use the docstring to provide this context if you don't think it's clear enough.

I do, however, like that you did shorten the name for the reason you listed.